packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -16,6 +16,9 @@ import 'debug.dart';
 import 'object.dart';
 
 /// How overflowing text should be handled.
+///
+/// A [TextOverflow] should be passed to a [RenderParagraph] widget via its
```

Comment:
Also: Why should we call out the RenderParagraph specifically? My guess is that this is most commonly used with the Text widget.

Reply:
You are right it has a lot more usages, I tried to gather them all in my update

Reply:
I would just list the most common use cases, which are Text and RichText. The other use cases are a little more esoteric and listing them seems to be more confusing than helpful.

Reply:
OK, I will keep `Text` and `RichText` references